routes/bot.js

```diff
@@ -6,16 +6,17 @@ import sortRandomly from '../helpers/sortRandomly.js'
 export default async function (fastify) {
   fastify.post(
     '/bot',
-    { onRequest: [fastify.zoom.verifyRequest] },
+    { preHandler: [fastify.zoom.verifyRequest] },
```

Comment:
why the change in the hook?

Reply:
the request body is not available in the `onRequest` hook: https://www.fastify.io/docs/latest/Reference/Hooks/#onrequest